src/main/java/com/bakdata/kafka/KafkaStreamsApplication.java

```diff
@@ -319,11 +320,11 @@ protected StateListener getStateListener() {
     protected void runCleanUp() {
         try (final ImprovedAdminClient adminClient = this.createAdminClient()) {
             final CleanUpRunner cleanUpRunner = CleanUpRunner.builder()
```

Comment:
@raminqaf I guess you never fixed the root cause of the issue because you need to check in the clean up runner if SR url is configured and then clean schemas or not https://github.com/bakdata/streams-bootstrap/blob/master/src/main/java/com/bakdata/kafka/util/ImprovedAdminClient.java#L60